### PR TITLE
drop py2, add django3

### LINKED ISSUE #25: Drop py2 support
Do what it says on the tin: py2 has been EOL for a while now. We should drop it.

### PR DESCRIPTION
**Pull request by costela**
This adds django3 testing and drops py2 support (fixes #25)

### CODE REVIEW

File: django_scrubber/migrations/0001_initial.py [hunk 1]
```diff
@@ -1,7 +1,5 @@
 # -*- coding: utf-8 -*-
```
Comment (MounirMesselmeni) [inline django_scrubber/migrations/0001_initial.py]:
```suggestion
```
I would remove all of those useless `# -*- coding: utf-8 -*-`

File: manage.py [hunk 2]
```diff
@@ -1,7 +1,5 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
```
Comment (MounirMesselmeni) [inline manage.py]:
```suggestion
```
Same here

File: runtests.py [hunk 3]
```diff
@@ -1,7 +1,5 @@
 #!/usr/bin/env python
 # -*- coding: utf-8
```
Comment (MounirMesselmeni) [inline runtests.py]:
```suggestion
```

File: django_scrubber/migrations/0001_initial.py (hunk 1 above)
Comment (lociii) [reply django_scrubber/migrations/0001_initial.py]: Agreed for "custom" files. I personally wouldn't change ones generated by Django like migrations.

File: manage.py (hunk 2 above)
Comment (lociii) [reply manage.py]: Also auto-generated

File: setup.py [hunk 4]
```diff
@@ -61,7 +61,9 @@ def get_version(*file_paths):
         "django-model-utils>=2.0,<4",
         "faker>=0.7.0,<3.0.0",
     ],
-    extras_require={":python_version<'3'": ["future>=0.16.0"]},
+    extras_require={
+        ":python_version<'3'": ["future>=0.16.0"],
```
Comment (lociii) [inline setup.py]: Can be removed when we don't support <3 anymore

File: tests/settings.py [hunk 5]
```diff
@@ -1,6 +1,4 @@
 # -*- coding: utf-8
```
Comment (MounirMesselmeni) [inline tests/settings.py]:
```suggestion
```

File: django_scrubber/migrations/0001_initial.py (hunk 1 above)
Comment (MounirMesselmeni) [reply django_scrubber/migrations/0001_initial.py]: Not the case anymore if you use Django 3

File: setup.py (hunk 4 above)
Comment (MounirMesselmeni) [reply setup.py]: Then maybe remove also python 2 from here also:
'Programming Language :: Python :: 2',
'Programming Language :: Python :: 2.7',

File: django_scrubber/migrations/0001_initial.py (hunk 1 above)
Comment (lociii) [reply django_scrubber/migrations/0001_initial.py]: Yup. But the project wasn't started with Django 3...

File: setup.py (hunk 4 above)
Comment (costela) [reply setup.py]: ah yes, of course! :heavy_check_mark:

File: django_scrubber/migrations/0001_initial.py (hunk 1 above)
Comment (costela) [reply django_scrubber/migrations/0001_initial.py]: removed in the non-generated files; left this one as is :heavy_check_mark:

**Comment by lociii** [comment]
Could you please remove the python 2 classifiers from setup.py?
Looks good otherwise. Thanks for your time and effort!

**Comment by costela** [comment]
> Could you please remove the python 2 classifiers from setup.py?
> Looks good otherwise. Thanks for your time and effort!

Done! :heavy_check_mark:

**Pull request merged**